django-s3-file-field-client/webpack.config.js

```diff
@@ -0,0 +1,39 @@
+const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  entry: "./src/client.ts",
+  output: {
+    path: path.resolve("build"),
+    filename: "joiist.js",
```

Comment:
What's up with these names?

Reply:
I was trying to determine what changes would or would not break the build. Apparently changing this does not.